### ISSUE TITLE
Django 1.8.6 BadMigrationError

### ISSUE THREAD

**Issue by jeroenweustink** [stefanfoulis/django-filer#609]
When running migrations I get a the exception BadMigrationError, I think it's due to the fact that django-filer 1.0.3 still contains south_migrations. I'm not able to fix it even when I remove the south migrations.

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 93, in handle
    executor = MigrationExecutor(connection, self.migration_progress_callback)
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/db/migrations/executor.py", line 19, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/db/migrations/loader.py", line 47, in __init__
    self.build_graph()
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/db/migrations/loader.py", line 182, in build_graph
    self.load_disk()
  File "/Users/jeroen/Projects/cms/venv/lib/python2.7/site-packages/django/db/migrations/loader.py", line 123, in load_disk
    "creating Django migrations." % app_config.label
django.db.migrations.loader.BadMigrationError: Migrated app 'filer' contains South migrations. Make sure all numbered South migrations are deleted prior to creating Django migrations.
```

**Comment by rense** [comment]
Looks like a merge error, South migrations are back in the directory "migrations". I tried removing those and moving the migrations from "migrations_django" to "migrations", but the "0002_auto_20150606_2003" is missing, which is a dependency for other migrations. I had to rollback and pin version 1.0.2 for now.

**Comment by jeroenweustink** [comment]
A rollback to 1.0.2 seems to work. The issue is explained here: https://code.djangoproject.com/ticket/25618

I will just stay at 1.0.2 for now.

**Comment by yakky** [comment]
Fixed in release 1.0.4, thanks for reporting

**Comment by jeroenweustink** [comment]
Thanks @yakky
